common/src/main/java/io/netty/util/internal/NativeLibraryLoader.java

```diff
@@ -125,10 +125,15 @@ public static void load(String originalName, ClassLoader loader) {
             }
         }
 
-        if (url == null) {
-            // Fall back to normal loading of JNI stuff
+        try {
+            // first try to load from java.library.path
             loadLibrary(loader, name, false);
+            logger.info("{} was loaded from java.libary.path", name);
```

Comment:
IMO log statements should be actionable.  In the case that it does load successfully from `java.library.path`, there isn't really much to do.  If it loads successfully but doesn't work right, that would warrant a log statement, which I think is what debug is good for.